src/ucp/core/ucp_rkey.c

```diff
@@ -20,16 +20,32 @@ static ucp_rkey_t ucp_mem_dummy_rkey = {
 static ucp_md_map_t ucp_mem_dummy_buffer = 0;
 
 
-ucs_status_t ucp_rkey_pack_uct(ucp_context_h context,
-                               ucp_md_map_t md_map, const uct_mem_h *memh,
-                               void *rkey_buffer, size_t *size_p)
+ucs_status_t ucp_rkey_packed_size(ucp_context_h context, ucp_md_map_t md_map)
```

Comment:
seems like return type should be `size_t`

Reply:
you are right, patch split issue...